14_functions_lab/01.py

```diff
@@ -1,7 +1,26 @@
-"""
-Write 2 functions:
-    mysum - returns the sum of its input arguments
-    mymul - returns the multiplication of its input arguments
-    Ignore non-numeric arguments
-"""
-
+"""
+Two fucntions: 1st is sum , 2nd is multiply.
+"""
+
+from types import *
+
+def sum(*args):
+    res = 0
+    for x in args:

```

Comment:
Better to use numbers.Number to check for numeric value, as that also covers floats:

```
import numbers

# True
print isinstance(10, numbers.Number)
# True
print isinstance(10.5, numbers.Number)

# False
print isinstance('bye', numbers.Number)
```